values.yaml

```diff
@@ -196,6 +212,7 @@ broker:
   <<: *env_anchor
   <<: *image_anchor
   image:
+    pullPolicy: Always
```

Comment:
The broker isn't a Thing, and instead of fixing it up we should remove this stanza, IMO.